### ISSUE TITLE
30.3 Update the frontend to comply with Web Content Accessibility Guidelines (WCAG)

### ISSUE THREAD

**Issue by PhilipVLam** [CDP-group-9/CDP_Trondheim_Kommune#113]
Acceptance Criteria: Ensure we comply with Trondheim Kommunes Graphical profile. Implement ARIA-labels for key components. In AI chat: Users can send a message by pressing 'ENTER'. In Checklist: User's should be able to move to next checkbox by pressing 'TAB' and crossing it out with 'ENTER' Guide: https://react-spectrum.adobe.com/react-aria/index.html

Resources: 6
Estimate: 1
